roles/monitoring/vars/main.yaml

```diff
@@ -16,6 +16,7 @@ logging_map:
     ES_LOGCOLLECTOR_PASSWD: "{{ V4M_LOGCOLLECTOR_PASSWORD }}"
     ES_METRICGETTER_PASSWD: "{{ V4M_METRICGETTER_PASSWORD }}"
     LOG_NS: "{{ V4M_LOGGING_NAMESPACE }}"
+    KB_KNOWN_NODEPORT_ENABLE: "{{ V4M_KB_KNOWN_NODEPORT_ENABLE }}"
```

Comment:
Again, use consistent names on these items KB is for Kabana and that's dead. Use dashboard or search as stated above.

Reply:
I don't disagree that we need to eventually change from KB or Kibana to OpenSearch dashboards, but other than names in the doc, no variable names have changed yet for this release. It doesn't seem to be a good idea to do some of them unless we do all of them. Up to this point, we have changed none of the existing variable names to reflect the switchover, although we are expecting to adopt such changes soon in an upcoming V4M release. Changing any of them at this point would likely break any customers using the current set of variables, which is what it seems Greg and team were hoping to avoid during this Kibana -> OpenSearch change.